system/lib/libcxxabi/src/cxa_guard.cpp

```diff
@@ -9,6 +9,29 @@
 
 #include "abort_message.h"
 
+#if __EMSCRIPTEN__
+
+// XXX EMSCRIPTEN making guard operations simple and LTO-optimizable opens up
```

Comment:
Not that I'm aware of. They seem to be an ABI type thing. I learned about them just from code.

Perhaps @sunfishcode knows more.